2.4/root/usr/bin/run-mongod

```diff
@@ -46,8 +32,8 @@ function cleanup() {
     mongo_remove
   fi
   echo "=> Shutting down MongoDB server ..."
-  if [ -f "${MONGODB_PID_FILE}" ]; then
-    kill -2 $(cat ${MONGODB_PID_FILE})
+  if [[ -s ${MONGODB_DATADIR}/mongod.lock ]]; then
+    kill -2 $(cat ${MONGODB_DATADIR}/mongod.lock)
```

Comment:
@rhcarvalho Do you know why `cleanup()` uses `kill` or `pkill` instead of `mongod --shutdown`?